game_eggs/minecraft/java/README.md

```diff
@@ -57,6 +57,10 @@ The second method requires you to know the id for both the modpack and version i
 may not get updated with the correct ip address and port at first launch.
 Please restart the server after first launch to fix this.**  
 
+## NanoLimbo
+[NanoLimbo](https://github.com/Nan1t/NanoLimbo)
+This is lightweight minecraft limbo server, written on Java with Netty. The main goal of the project is maximum simplicity with a minimum number of sent and processed packets.
```

Comment:
Please correct this grammar

("This is a lightweight" or preferably "A lightweight")